crates/lang/src/env_access.rs

```diff
@@ -35,12 +37,12 @@ use ink_eth_compatibility::ECDSAPublicKey;
 /// This allows ink! messages to make use of the environment efficiently
 /// and user friendly while also maintaining access invariants.
 #[derive(Copy, Clone)]
-pub struct EnvAccess<'a, T> {
-    /// Tricks the Rust compiler into thinking that we use `T`.
-    marker: PhantomData<fn() -> &'a T>,
+pub struct EnvAccess<'a, E> {
```

Comment:
I get why you renamed `T` to `E`, but in the future can you at least split this out into its own commit? It adds a lot of unnecessary noise. Tbh it should've been done in a follow-up PR